src/coreclr/jit/llvm.cpp

```diff
@@ -98,6 +97,15 @@ extern "C" DLLEXPORT void registerLlvmCallbacks(void*       thisPtr,
     }
 }
 
+GCInfo* Llvm::getGCInfo()
+{
+    if (_gcInfo == nullptr)
+    {
+        _gcInfo = new GCInfo(_compiler);
```

Comment:
This should use the compiler's allocator to avoid leaking: `new (_compiler->getAllocator(CMK_GC)) GCInfo(_compiler);`